### DISCUSSION TITLE
Way to show SSR indicator for tab Indicators?

### DISCUSSION THREAD

**Discussion by howagain** [kobaltedev/kobalte#175] [Q&A]
[Tabs](https://kobalte.dev/docs/core/components/tabs) seem to use JS to show the indicator. This means there's an annoying visual bug of the indicator popping into existence. Is there a way to use CSS or some other non-js way to show what is selected.

**Comment by fabien-ml** [comment]
Hi, sorry I haven't had time to go through this topic. Yes `Tabs.Indicator` uses JS for measurement and to do the translate animation.

Since the indicator is only for presentation and doesn't have a11y logic you can build your own and show it with css.

For example you can add a border to `Tabs.Trigger` when it have the `data-selected` attribute.
